opentech/public/people/templates/people/includes/person_listing.html

```diff
@@ -1,7 +1,13 @@
-<article class="person">
-    <h2><a href="{% pageurl person %}">{{ person.first_name }} {{ person.last_name }}</a></h2>
-
+{% load wagtailcore_tags wagtailimages_tags %}
```

Comment:
I would usually leave a blank line between load or extend tags and the template content, just for readability. Not mandatory though